using_images/db_images/mongodb.adoc

```diff
@@ -191,35 +222,33 @@ $ oc env dc mongodb \
   MONGODB_ADMIN_PASSWORD=<new_admin_password>
 ----
 
+[IMPORTANT]
+====
+If multiple deployment configurations utilize these environment variables, for
+example in the case of an application created from a template, you must perform
+the above task on each deployment configuration so that the passwords are in
+sync.
+====
```

Comment:
@adellape your finding is true, however I'd suggest doing the changes at once instead of one at a time.

See the example [here](https://github.com/openshift/postgresql/tree/master/examples/replica#postgresql_master_password), you can change the env for multiple DCs in one `oc env` call.